packages/next/errors.json

```diff
@@ -626,5 +626,9 @@
   "625": "Server Actions are not supported with static export.\\nRead more: https://nextjs.org/docs/app/building-your-application/deploying/static-exports#unsupported-features",
   "626": "Intercepting routes are not supported with static export.\\nRead more: https://nextjs.org/docs/app/building-your-application/deploying/static-exports#unsupported-features",
   "627": "cacheLife() is only available with the experimental.useCache config.",
-  "628": "cacheTag() is only available with the experimental.useCache config."
+  "628": "cacheTag() is only available with the experimental.useCache config.",
+  "629": "Invariant: Module is recorded after the report is generated. This is a Next.js internal bug.",
```

Comment:
nit: while they are invariants, I still liked that the old errors pointed to the offending plugin. maybe we can bring back the `[SlowModuleDetection]` prefix so folks know to disable it if it's causing issues